app/api/currentuser/delete.go

```diff
@@ -0,0 +1,55 @@
+package currentuser
+
+import (
+	"errors"
+	"net/http"
+
+	"github.com/go-chi/render"
+
+	"github.com/France-ioi/AlgoreaBackend/app/loginmodule"
+	"github.com/France-ioi/AlgoreaBackend/app/service"
+)
+
+// swagger:operation DELETE /current-user users currentUserDeletion
+// ---
+// summary: Delete the current user
+// description: Deletes all information stored in the platform related to the current user
+//              and calls the “UnlinkClient” service of the login module
+//
+//              * The deletion is rejected if the user is a member of at least one group with
+//              `now() < lockUserDeletionDate`
```

Comment:
It should be added to the description of the previous "delete temp user" command as well.

Reply:
The command doesn't check the lockUserDeletionDate at all (because it wasn't mentioned in #186). Should it?

Reply:
Anyway, that is a tool for admins... forget about it :-)